### ISSUE TITLE
Suggestion -> Config should be renamed

### ISSUE THREAD

**Issue by Varkal** [OneSignal/emoji-picker#64]
Hello, 

I lose a couple of hours to understand why your library doesn't works with my project..... until i understand you use a class named Config... override by my Config class (same name...)

Temporary I will rename my Config class, but, maybe you should rename your Config class in something like EmojiPickerConfig to limit probability of collision ?

**Comment by chrispcb** [comment]
All scripts should be merged and minified on one file like dist/emojipicker.min.js.

**Comment by Varkal** [comment]
Don't really understand connection between the presence of a minified build, and the name of a class/function/variable in the code... 

Unless you suggest the name mangling during minification is enough... 
If yes, I totally disagree -> No reason to use minified build in development : I want be capable of reading code of library and eventually put breakpoint into it to understand what happen.

**Comment by chrispcb** [comment]
Yes, I agree with you. I meant to create a folder "/dist" which would contain 1 file and the same in minified.
